lotto645.py

```
@@ -63,98 +67,124 @@ def buy_lotto645(
+            "saleMdaDcd": "10"
         }
 
     def _generate_body_for_manual(self, cnt: int) -> dict:
-        assert type(cnt) == int and 1 <= cnt <= 5
-
+        assert isinstance(cnt, int) and 1 <= cnt <= 5
         raise NotImplementedError()
 
-    def _getRequirements(self, headers: dict) -> list: 
+    def _getRequirements(self, headers: dict) -> list:
         org_headers = headers.copy()
```

Ruff (0.14.10)

Local variable `org_headers` is assigned to but never used

Remove assignment to unused variable `org_headers`

(F841)